features/step_definitions/validates_coffeescript_style_guide_steps.rb

```diff
@@ -0,0 +1,14 @@
+def executing_coffee_lint
+  "executing 'coffeelint .'"
+end
+
+def coffee_lint_output
+  raw_output = command_output_for_project_or_file('coffeelint.rb -r .')
+  raw_output.split("\n").map do |line|
+    if line.match(/✗ #\d+:/)
+      line.gsub("\e[32m", '').gsub("\e[0m", '').gsub('✗', 'ERROR:')
+    else
+      ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.